src/ql/arch/cc/info.cc

```diff
@@ -178,7 +179,7 @@ void Info::dump_docs(std::ostream &os, const utils::Str &line_prefix) const {
                 {   "type": "mw",
                     "operand_idx": 0,
                     "value": [
-                        "{gateName}-{instrumentName}:{instrumentGroup}-gi",
+                        "{gateName}-{instrumentName}:{instrumentGroup}-gi",  // FIXME: outdated
```

Comment:
Also this FIXME.

Reply:
The documentation is lagging behind a bit, will fix when things have stabilized